c_src/ep_encoder.c

```diff
@@ -736,9 +738,9 @@ pack_element_packed(ErlNifEnv *env, ERL_NIF_TERM term, ep_enc_t *enc, ep_field_t
 
     case field_int64:
         if (field->ebin == TRUE) {
-            check_ret(ret, pack_bin_as_int64(env, term, enc));
+            check_ret(ret, pack_bin_as_int64(env, term, enc, field));
         } else {
-            check_ret(ret, pack_int64(env, term, enc));
+            check_ret(ret, pack_int64(env, term, enc, field));
```

Comment:
did something change in these functions?

Reply:
yes, i think they've changed the logic for when to not encode zero fields